### PR TITLE
OCSADV-296: Bug fix: only look at last part in data label for index

### PR DESCRIPTION
**Pull request by fnussber**
Small bug fix.

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala [hunk 1]
```diff
@@ -92,13 +92,19 @@ object ItcUniqueConfig {
...
     findSpans(ls).map {
       case i :: Nil => i.toString
       case i :: is => f"$i%03d-${is.last}%03d"
     }.mkString(", ")
   }
 
+  // Only the number at the very end is relevant
+  private def labelIndex(c: Config): Int = {
+    val label = c.getItemValue(DATALABEL_KEY).asInstanceOf[String]
+    label.split('-').last.toInt
```
Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala]: I think this may NPE if there is no "-" on the string

Comment (fnussber) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala]: This will return an empty list if there is no ```'-'```.

Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala]: I think `last` on an empty list will throw an exception

Comment (fnussber) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala]: Even for an empty string ```split()``` does not return an empty array but an array with an empty string:

```scala
scala> "".split('-')
res0: Array[String] = Array("")

scala> "123".split('-')
res1: Array[String] = Array(123)

scala> "123-123".split('-')
res2: Array[String] = Array(123, 123)
```

Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala]: my bad :+1:

**Pull request merged**